cmd/mesh/root.go

```diff
@@ -24,9 +24,7 @@ import (
 )
 
 const (
-	SetFlagHelpStr = `Set a value in IstioOperator CustomResource. e.g. --set policy.enabled=true.
-Overrides the corresponding path value in the selected profile or passed through IstioOperator CR
-customization file`
+	SetFlagHelpStr = `Can be used to choose a profile (--set profile=demo), enable or disable components (--set policy.enabled=true), or override Istio settings(--set values.global.mtls.enabled=true).  See documentation for more info: https://istio.io/docs/reference/config/istio.operator.v1alpha12.pb/#IstioControlPlaneSpec`
```

Comment:
Generally --set is used to override a path in IstioOperator API's spec, inlcluding profile, component, and values. We need one sentence to summarize that first.

Reply:
Good point, I've added some verbiage to reflect that in a single sentence before going into the examples